src/components/comments/Answer.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function Answer(props) {
+  return (
+    <div className="commentBubble answerBubble">
+      {/* Puts the answer header and answer together within the same bubble*/}
+      <div className="boldComment">{props.answerHeaderText}</div>
+      <div className="comment">{props.answerText}</div>
```

Comment:
I think having either only `answerText` or `answerHeaderText` for `Answer` components can make it so that our answers can have just bold text or just unbolded text, what issues can we run into down the line using this approach?